workflows/rules/variantcalling/manta.smk

```diff
@@ -5,82 +5,89 @@ from workflows.scripts.annotate_manta.manta_summary import manta_summary
 
```

Comment:
I assume this one is for hg19 since we have manta_hg38. Could be useless

Reply:
Yes, several rules are divided into hg19/hg38. As far as I can tell, it should be possible to support both, but that will be a separate PR.

Reply:
I would say dont spend time on it now. No one is using hg19 for somatic in routine and we have more important eggs to fry